Simplenote/AuthViewController+Swift.swift

```diff
@@ -6,6 +6,10 @@ extension AuthViewController {
 
     @objc
     func setupInterface() {
+        if state == nil {
+            state = AuthenticationState()
+        }
```

Comment:
We can just do this in `AuthViewController.m`'s init method:

```
        self.state = [AuthenticationState new];
```

Reply:
done